docs/plugins/filters/jdbc_static.asciidoc

```diff
@@ -6,9 +6,9 @@
 ///////////////////////////////////////////
 START - GENERATED VARIABLES, DO NOT EDIT!
 ///////////////////////////////////////////
-:version: v5.4.1
-:release_date: 2022-11-04
-:changelog_url: https://github.com/logstash-plugins/logstash-integration-jdbc/blob/v5.4.1/CHANGELOG.md
+:version: v5.4.4
+:release_date: 2023-06-15
+:changelog_url: https://github.com/logstash-plugins/logstash-integration-jdbc/blob/v5.4.4/CHANGELOG.md
```

Comment:
Doc changes added in [#122](https://github.com/logstash-plugins/logstash-integration-jdbc/pull/122) and picked up in `#125`